multi-transfer-payments/client-completed.js

```diff
@@ -0,0 +1,81 @@
+const IlpPacket = require('ilp-packet')
+const HostedLedgerPlugin = require('ilp-plugin-payment-channel-framework')
+const plugin = new HostedLedgerPlugin({
+  server: 'btp+ws://:@localhost:9000/'
+})
+const uuid = require('uuid/v4')
+const fetch = require('node-fetch')
+const crypto = require('crypto')
+
+const connectorAddress = process.argv[2]
+
+function base64url (buf) {
+  return buf.toString('base64')
+    .replace(/\+/g, '-').replace(/\//g, '_').replace(/=+$/, '')
+}
+
+function sha256 (preimage) {
+  return crypto.createHash('sha256').update(preimage).digest()
+}
+
+function hmac (secret, input) {
+  return crypto.createHmac('sha256', secret).update(input).digest()
+}
+
+plugin.connect().then(function () {
```

Comment:
Nitpick: This could be an anonymous async function using async/await to improve readability.

Reply:
the tutorial reader doesn't necessarily know how async/await works, only [JS + Promises + Buffers](https://github.com/interledger/tutorials/pull/1/files#diff-2e7846f7c5f5ad0a54cddbb274126991R9)